puppet/zulip/manifests/tornado_sharding.pp

```diff
@@ -13,7 +13,7 @@
     group   => 'root',
     mode    => '0644',
     notify  => Service['nginx'],
-    content => "set \$tornado_server http://tornado;\n",
+    content => "# Configuration hash: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nset \$tornado_server http://tornado;\n",
```

Comment:
Where does this hash come from?

Reply:
What not everyone has memorized the sha256 empty-string hash? :) I'll fix that with a comment.